### DISCUSSION TITLE
Proper Way To Delete Full Pcap?

### DISCUSSION THREAD

**Discussion by GeorgiaM-honestly** [Security-Onion-Solutions/securityonion#7610] [Unsupported Versions] [answered]
Hello! I am running a new install using the official installer of version 2.3.110. I'm running inside of a virtual machine. I make frequent image/disk level backups using clonezilla. I love full pcap however it causes the backup (which uses varying types and levels of compression) to take a lot longer, and to take more space. My typical approach in FORMER installs has been to simply manually delete everything in /nsm/pcap or even hack things up and add another "disk" for /nsm/pcap and then not back up that disk in clonezilla.

Is there a "proper" way to do this? I know that I can limit how much space it will take up, however, I don't mind occasionally ditching the full pcap files as a tradeoff for faster / smaller backups. My concern is that this will goof up something internally with steno or other tools.

**Comment by dougburks** [comment, marked as answer]
The supported method to delete full packet capture is to ask steno to do it via configuration:
https://docs.securityonion.net/en/2.3/stenographer.html#configuration

There might be other ways, but they would be untested and unsupported.
